css/style.css

```diff
@@ -43,6 +43,115 @@ Developer @GA in NYC
 /*******************************************/
 
 body {
+  font-family: 'Open Sans', sans-serif;
+}
+header {
+  background-color: #033048;
+  padding: 20px 50px;
+}
+article {
+  float: left;
+  width: 700px;
+  padding: 50px 20px 50px 50px;
+  box-sizing: border-box;
+}
+aside {
+  float: left;
+  width: 240px;
```

Comment:
looks like you're defining a width here and then redefining it below with your `.aside` class which is overriding the `aside` width style. It works out fine in your design but it probably makes more sense to stick with consistent units when defining widths for your big containers. Your article was set to `700px` but then your `.sidebar` was set to `25%`. Its okay to use percentages for some things, pixels for others. But when in this case, it'd be better to stick to consistent units for `width`s